at-luantran/Week5/migrate/20180717095550_change_gender_to_be_int_in_users.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeGenderToBeIntInUsers < ActiveRecord::Migration[5.2]
+  def up
+    User.where(gender: 'nam').update_attribute(gender: '1')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/df604cafc3fe878bc15e3cecf58af95294f22919/at-luantran/Week5/migrate/20180717095550_change_gender_to_be_int_in_users.rb#L3"></span>Rails/SkipsModelValidations: Avoid using <code>update_attribute</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>